css-canvas-medical/styles.css

```diff
@@ -0,0 +1,239 @@
+.container {
+  height: 120rem;
+}
+
+.row {
+  display: flex;
+  height: 65px;
+  border-bottom: 1px solid rgb(163, 163, 163);
+}
+
+h1 {
+  padding-left: 105px;
+  font-weight: 300;
+  font-size: 45px;
+  margin-right: 370px;
+}
+
+.canvas {
+  display: flex;
+  align-items: center;
+  color: rgb(9, 9, 97);
+}
+
+.directory {
+  display: flex;
+  align-items: center;
+}
+
+button {
+  border: none;
+  background-color: white;
+  margin-left: 40px;
+  color: grey;
+  font-family: sans-serif, serif;
+}
+
+.contact-us {
+  border: 1px solid rgb(8, 8, 73);
+  border-radius: 6%;
+  background-color: rgb(8, 8, 73);
+  color: white;
+  padding: 10px;
+}
+
+.row-image {
+  margin-top: 65px;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 50px;
+}
+
+.image {
+  width: 1050px;
+  height: 500px;
```

Comment:
use either a height or a width on images or it will stretch
if you want it to be a container you can set it as a background image of a div.